kubernetes/omsagent-otel.yaml

```diff
@@ -691,25 +694,28 @@ spec:
     metadata:
       labels:
         app: opentelemetry
-        component: otel-collector
+        component: omsagent-otel
     spec:
       containers:
-      - name: otel-collector
-        image: ayushisingh/omsagent:07162020
+      - name: omsagent-otel
+        image: ayushisingh/omsagent:08032020v2
         command:
           - "/otelcontribcol"
-          - "--config=/conf/otel-collector-config.yaml"
+          - "--config=/conf/omsagent-otel-config.yaml"
         resources:
           limits:
-            cpu: 1
-            memory: 2Gi
+            cpu: 150m
+            memory: 600Mi
           requests:
-            cpu: 200m
-            memory: 400Mi
+            cpu: 75m
+            memory: 225Mi
+        env: 
+          - name: APPLICATIONINSIGHTS_AUTH
```

Comment:
we should put this in dockerfile similar to what we do in container insighsts agent.